### ISSUE TITLE
ssl_verify argument not used in get_status_page()

### ISSUE THREAD

**Issue by jmcmillan89** [lucasheld/uptime-kuma-api#65]
When instantiating an instance of the `UptimeKumaApi` class, you can disable SSL verification via the `ssl_verify` argument in the constructor:
```python
def __init__(
    self,
    url: str,
    timeout: float = 10,
    headers: dict = None,
    ssl_verify: bool = True,
    wait_events: float = 0.2
 ) -> None:
```
This is then set on the Client: `self.sio = socketio.Client(ssl_verify=ssl_verify)` so any web requests issued by the Client use this variable.

However, there is a single `requests.get()` within `get_status_page()` that bypasses the Client, so the `ssl_verify` arg is not passed:
```python
try:
    r2 = requests.get(f"{self.url}/api/status-page/{slug}", timeout=self.timeout).json()
except requests.exceptions.Timeout as e:
    raise Timeout(e)
```

As a result, if you instantiate your API with `api = UptimeKumaApi("https://localhost:8080", ssl_verify=False)`, calling `api.get_status_page()` will fail with an SSL verification error.

**Comment by pr0kium** [comment]
+1

**Comment by pbarone** [comment]
This is right, and @pr0kium's PR #81 has the fix - `ssl_verify` was passed to the socket.io client but never applied to the `requests.get` behind `get_status_page`, so a self-signed endpoint failed regardless of the setting. With the v2 PRs here sitting unreviewed, I carried the library forward as a continuation, `uptime-kuma-api2`, and applied it in 2.3.0 (`verify=self.ssl_verify` on the status-page fetch), with @pr0kium credited. It's verified against a genuinely untrusted certificate: with `ssl_verify=False` the read succeeds, with `ssl_verify=True` the same read raises `SSLError`.

`pip install uptime-kuma-api2` - same `uptime_kuma_api` import package, original MIT license and @lucasheld's copyright retained. Full context in #75. Happy to send changes back upstream if this project becomes active again.
